htsohm/binning.py

```diff
@@ -33,10 +38,14 @@ def select_parents(run_id, children_per_generation, generation):
     for child in next_generation:
         # First, the bin is selected...
         parent_bin = np.random.choice(bins, p=weights)
-        potential_parents = [i[0] for i in session.query(Material.id).filter(
-            Material.run_id == run_id, Material.methane_loading_bin == parent_bin["ML"],
-            Material.surface_area_bin == parent_bin["SA"],
-            Material.void_fraction_bin == parent_bin["VF"]).all()]
+        parent_query = session \
+            .query(Material.id) \
+            .filter(
+                Material.run_id == run_id, Material.methane_loading_bin == parent_bin["ML"],
```

Comment:
not to be nit-picky, but since they're all conditions I'd probably separate this out into another line, i.e.
 Material.run_id == run_id,
 Material.methane_loading_bin == parent_bin["ML"]
otherwise when you glance at that block, you think there are three conditions, but there really are four.